components/Dashboard.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <div>
```

Comment:
Probably don't need this wrapper `<div>` for now since this component is already self-contained in a `<header>`.